include/lemlib/logger/baseSink.hpp

```diff
@@ -138,14 +162,43 @@ class BaseSink {
         virtual void sendMessage(const Message& message);
 
         /**
-         * @brief Set the format of the logger
+         * @brief Set the format of messages that the sink sends
          *
          * @param format
+         *
+         * Changing the format of the sink changes the way each logged message looks. The following named formatting
+         * specifiers can be used:
+         * - {time} The time the message was sent in milliseconds since the program started.
+         * - {level} The level of the logged message.
+         * - {message} The message itself.
+         *
+         * <h3> Example Usage </h3>
+         * @code
+         * infoSink()->setFormat("[LemLib] -- {time} -- {level}: {Message}");
+         * infoSink()->info("hello");
+         * // This will be formatted as:
+         * // "[LemLIb] -- 10 -- INFO:  hello"
+         * @endcode
          */
         void setFormat(const std::string& format);
 
         /**
-         * @brief Get the arguments for formatting
+         * @brief Get the extra named arguments for formatting
+         *
+         * Can be overridden to add extra named arguments to the sink's format.
+         * <h3> Example Usage </h3>
+         *
+         * The following code would add a {zero} formatting argument which could be used in setFormat method of this
+         * sink.
+         *
+         * @code
+         * fmt::dynamic_format_arg_store<fmt::format_context>
+         * ExampleSink::getExtraFormattingArgs(const Message& messageInfo) {
+         *     fmt::dynamic_format_arg_store<fmt::format_context> args;
+         *       args.push_back(fmt::arg("zero", 0);
+         *       return args;
```

Comment:
```suggestion
         *      args.push_back(fmt::arg("zero", 0);
         *      return args;
```
The spacing here should be fixed